src/mixins/common-mixin.ts

```diff
@@ -86,53 +86,23 @@ export default class CommonMixin extends Vue {
 
   /** The appropriate edit label for corrections, alterations, change or conversion filings. */
   get editLabel (): string {
-    // if (this.isCorrectionFiling) return 'Correct'
-    // if (this.isAlterationFiling || this.isChangeFiling || this.isConversionFiling) return 'Change'
-    // return 'Edit'
-    switch (true) {
-      case this.isCorrectionFiling:
-        return 'Correct'
-      case this.isAlterationFiling:
-      case this.isChangeFiling:
-      case this.isConversionFiling:
-        return 'Change'
-      default:
-        return 'Edit'
-    }
+    if (this.isCorrectionFiling) return 'Correct'
```

Comment:
I reverted this code as I thought the original code was more clear.

But, still, the `switch (true)` syntax might be useful for something in the future.